src/main/resources/twitter-demo/static/js/tweetbook.js

```diff
@@ -755,7 +755,15 @@ function drawHashtag(tag_count) {
  **/
 function drawTweets(message) {
   $.each(message, function (i, d) {
-    $('#tweets tr:last').after('<tr><td>' + d.uname + '<br/>' + d.tweet + '</td></tr>');
+    // Using JSONP
+    var url = "https://api.twitter.com/1/statuses/oembed.json?id=" + d.id;
+    $.ajax({
```

Comment:
Got it. It seems not hard. We should put this request into the server(that python script), and only send the $.ajax to our own server.